README.md

```diff
@@ -7,36 +7,33 @@
 ![Python 3.12](https://img.shields.io/badge/Python-3.12-blue.svg)
 [![Docker Image Version (latest semver)](https://img.shields.io/docker/v/safeglobal/safe-cli?label=Docker&sort=semver)](https://hub.docker.com/r/safeglobal/safe-cli)
 
-# Safe-CLI
+# Safe CLI
 
-Command line utility for **Safe** contracts. Use it to manage your **Safe** easily from the command line.
+Safe CLI is a command-line utility for Safe contracts. You can use it to manage your Safe Smart Account from the command line.
 
-Safe-CLI does not rely on _Safe{Core} API_, so it can be used in networks where Safe services are not available. If they are available,
-Safe-CLI can also interact with them in `tx-service` mode.
+It does not rely on Safe{Core} API and can also be used in networks where Safe services are unavailable. Learn more through the [documentation](https://docs.safe.global/advanced/cli-overview).
```

Comment:
Why do you remove the bolding and italic?

Reply:
This has been copied from the docs, keeping them in line with the doc's style guide.